Tests.Integration.Presentation.Web/Authorization/GetOrganizationsTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.Organization;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Authorization
+{
+    public class GetOrganizationsTests : WithAutoFixture
+    {
+        [Theory]
+        [InlineData("Name")]
+        [InlineData("")]
+        [InlineData(null)]
+        public async Task Get_Organizations_Returns_List(string orderBy)
+        {
+            var response = await OrganizationHelper.GetOrganizationsResponseAsync(orderBy);
+
+            Assert.True(response.IsSuccessStatusCode);
+        }
+
+        [Fact]
+        public async Task Get_Organizations_Does_Not_Allow_For_Property_Other_Than_Name()
+        {
+            var incorrectOrderBy = "IncorrectPropertyName";
+
+            var response = await OrganizationHelper.GetOrganizationsResponseAsync(incorrectOrderBy);
+
+            Assert.True(!response.IsSuccessStatusCode);
```

Comment:
please verify the error code you expect. 500 would be unacceptable, but 400 is to be expected